collective/pwexpiry/locales/collective.pwexpiry.pot

```diff
@@ -38,6 +62,18 @@ msgstr ""
 msgid "Passwords must contain at least three of the following four character groups: Uppercase characters (A through Z), Lowercase characters (a through z), Numerals (0 through 9), Special characters such as !, $, #, %"
 msgstr ""
 
+#: ./collective/pwexpiry/skins/pwexpiry/login_form_validate.vpy:48
+msgid "Please enter your email address."
```

Comment:
is there really a need to translate these too? can't we re-use msgids from i18n-domain plone here?